packages/client/hmi-client/src/workflow/util.ts

```diff
@@ -47,26 +47,32 @@ export const chartActionsProxy = (node: WorkflowNode<any>, updateStateCallback:
 	};
 };
 
-export const getTimespan = (dataset?: CsvAsset, mapping?: CalibrateMap[]): TimeSpan => {
+export interface GetTimespanParams {
+	dataset?: CsvAsset;
+	mapping?: CalibrateMap[];
+	timestampColName?: string;
+}
+
+export const getTimespan = (params: GetTimespanParams): TimeSpan => {
```

Comment:
It doesn't need out of scope awareness, a function can be just fine here.

```suggestion
export function getTimespan(params: GetTimespanParams): TimeSpan {
```